cmd/ak/cmd/users/get_id.go

```diff
@@ -0,0 +1,30 @@
+package users
+
+import (
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+)
+
+var getIDCmd = common.StandardCommand(&cobra.Command{
```

Comment:
What's the point of this command, compared to a regular "get"?

Reply:
does not return the entire user, just the user id. don't want people to be able to see user details they're not allowed to see.

Reply:
the point is this: if I'm not allowed to see your details, why should I be allowed to see your user ID?